rules/validator_options.go

```diff
@@ -56,6 +56,18 @@ type ValidatorOptions interface {
 	// RequiredRollupTags returns the list of required rollup tags.
 	RequiredRollupTags() []string
 
+	// SetRollupTagInvalidChars sets the list of invalid chars for a rollup metric tag.
+	SetRollupTagInvalidChars([]rune) ValidatorOptions
```

Comment:
I'd just call this `SetTagNameInvalidChars` since they apply to the tags of all metrics. We could use them to validate the tags in the tag filters for both mapping rules and rollup rules.

Reply:
Yeah that sounds much better, done.